pkg/docker/config/config.go

```
@@ -646,6 +665,12 @@ func findAuthentication(ref reference.Named, registry, path string, legacyFormat
 		return getAuthFromCredHelper(ch, registry)
 	}
 
+	if auths.CredsStore != "" {
+		if cred, err := getAuthFromCredHelper(auths.CredsStore, registry); err == nil {
```

Why is it OK to completely ignore errors here?